src/wavm/openmp.cpp

```diff
@@ -153,6 +154,20 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
     return ctx.threadNumber;
 }
 
+// ------------------------------------------------
+// TIMING
+// ------------------------------------------------
+
+WAVM_DEFINE_INTRINSIC_FUNCTION(env, "omp_get_wtime", F64, omp_get_wtime)
```

Comment:
This was added in `faasm/cpp` and needed to be put here too. It's tested via a function in that repo.